build/publish/AzureInfo.targets

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+   
+  <Target Name="SetAzureProps"
+          Condition=" '$(IsOrchestratedPublish)' == 'True' And '$(PB_PublishBlobFeedUrl)' != '' ">
+
+    <ParseBlobUrl BlobUrl="$(PB_PublishBlobFeedUrl)">
+      <Output TaskParameter="BlobElements" ItemName="BlobElements" />
+    </ParseBlobUrl>
+
+    <PropertyGroup>
+      <PublishBlobFeedUrlBlobPath>%(BlobElements.BlobPath)</PublishBlobFeedUrlBlobPath>
+    </PropertyGroup>
+
+    <RegexReplace Input="$(PublishBlobFeedUrlBlobPath)" Expression="index.json" Replacement="" Count="1">
```

Comment:
This can just be done with $(PublishBlobFeedUrlBlobPath.Replace('/index.json', ''))

Reply:
Nice, much better....